modules/ivanov_n_greatest_inc_subseq/src/subseq.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 Ivanov Nikita
+
+#include "include/subseq.h"
+
+void LIS::addNumber(int a) {
+    nums.push_back(a);
+}
+
+size_t LIS::getSize() {
+    return nums.size();
+}
+
+std::vector<int> LIS::run() {
+    size_t n = nums.size();
```

Comment:
replace with internal function
```suggestion
    size_t n = getSize();
```